### PR TITLE
Add PV to spiderfoot chart

### PR DESCRIPTION
**Pull request by thamudi**

### CODE REVIEW

**Review by thread-koder — CHANGES_REQUESTED** [review]
Cannot merge this PR as the chart is not passing the helm chart lint, 
will merge when the chart passes the helm lint.

**Comment by thamudi** [comment]
Closing this. Fixed in #39

**Pull request closed without merging**
